### DISCUSSION TITLE
Javascript

### DISCUSSION THREAD

**Discussion by farhanbaeee** [community/community#80304] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Javascript or Go?

**Comment by farhanbaeee** [comment]
Javascript or go? react or viu?

**Reply by fmpangestu** [reply]
me javascript, and react

**Reply by jane-does-coding** [reply]
Javascript, and React are most popular

**Comment by farhanbaeee** [comment]
express or next?

**Reply by fmpangestu** [reply]
me next

**Reply by jane-does-coding** [reply]
Express for starting and after fluent in express move on to next.js

**Comment by farhanbaeee** [comment]
give me step frontend javascript

**Reply by fmpangestu** [reply]
easy, your consistents

**Comment by Develops01** [comment]
your question is totally wrong
these are only tools
you should look what you want to do
then select a tool
js or go? 
it has no meaning

**Reply by fmpangestu** [reply]
me js, you js or go?

**Comment by ashutosh-naik** [comment]
JavaScript is more popular than Golang

**Comment by ghostinhershell** [comment]
Hi @farhanbaeee, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by sayyam267** [comment]
Javascript
